test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py

```diff
@@ -183,3 +195,14 @@ def sortBy(self, headerText):
                 }
             )
         )
+
+    def OWNCLOUD_AS_A_ROOT_SYNC_DIRECTORY(self, folderName):
```

Comment:
```suggestion
    def SELECT_A_ROOT_SYNC_DIRECTORY(self, folderName):
```
Because you will choose folderName as root sync dir. It might not always be "ownCloud"